modules/javafx.graphics/src/main/java/com/sun/glass/ui/Application.java

```diff
@@ -800,4 +800,10 @@ public Map<String, String> getPlatformKeyMappings() {
     public Map<String, Class<?>> getPlatformKeys() {
         return Map.of();
     }
+
+    /**
+     * Checks whether there are any problems with platform preferences detection,
+     * and emits a warning otherwise.
```

Comment:
Suggestion: change "otherwise" to "if so" or similar (the current comment indicates that a warning will be produced if there are no problems).